.github/workflows/deploy.yml

```
@@ -0,0 +1,54 @@
+name: Deploy to GitHub Pages
+
+on:
+  push:
+    branches: ['main']
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+concurrency:
+  group: 'pages'
+  cancel-in-progress: true
+
+jobs:
+  deploy:
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: 'npm'
+
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v2
+        with:
+          version: 8
```

이건 잘 모르긴 해도 버전을 최신 버전으로 바꿔 보시면 될지도...?

```suggestion
      - name: Setup pnpm
        uses: pnpm/action-setup@v4
        with:
          version: 10
```
https://pnpm.io/continuous-integration

---

pnpm CI 문서 참고하여 pnpm 및 action-setup 버전 업그레이드했습니다! 감사합니다! f6cf84a3b72102ab665190f8307a1e61f96b22bf